LDMP/calculate.py

```diff
@@ -65,6 +67,39 @@ class tr_calculate(object):
     def tr(message):
         return QCoreApplication.translate("tr_calculate", message)
 
+# set mapping among GUI interface and algorithm group. Can be done via a JSON configuration as in scripts.py
+# bu algorithms are almost stable (as in alg list in main widget tab) => leaved as static dictionary
+local_scripts = {}
+local_scripts['DlgCalculateLDNSummaryTableAdmin'] = {
+    'group': 'SDG 15.3.1',
+    'display_name': 'Final SDG 15.3.1 - Summary Table',
+    'source': 'LDNSummaryTable'
+}
+local_scripts['DlgCalculateUrbanSummaryTable'] = {
+    'group': 'SDG 11.3.1',
+    'display_name': 'Urban Summary Table',
+    'source': 'UrbanSummaryTable'
+}
+local_scripts['DlgCalculateRestBiomassSummaryTable'] = {
+    'group': 'Potencial change in biomass',
+    'display_name': 'Rest Biomass Summary Table',
+    'source': 'RestBiomassSummaryTable'
+}
+local_scripts['DlgCalculateLC'] = {
+    'group': 'SDG 15.3.1',
+    'display_name': 'Land cover (locally)',
+    'source': 'CalculateLC'
+}
+local_scripts['DlgCalculateTCData'] = {
+    'group': 'total-carbon',
+    'display_name': 'Total Carbon (locally)',
+    'source': 'CalculateTCData'
+}
+local_scripts['DlgCalculateSOC'] = {
+    'group': 'SDG 15.3.1',
+    'display_name': 'Soil Organic Carbon (locally)',
+    'source': 'CalculateSOC'
+}
```

Comment:
Still I'll be best to have these defined in a separate file e.g. `data/scripts.json` and maybe add a `mode` property that will have `local` or `remote` values.

Reply:
I can do it, but I followed the same criteria as expressed times ago by @ricardogsilva regard the algorithm list configuration  file... no need to have an external code file because alg list is almost static => I didn't spent time to create conf file related with alg list.

Reply:
> and maybe add a `mode` property that will have `local` or `remote` values.

adding a `mode` means change the actual workflow of processing e.g. change call to local or remote basing on this value e.g. more changes in calculate_xx.py basing on this value. I preferred to leave the workflow as before.

Reply:
I meant the `mode` property should be defined the in the file not on the class, same way the `group`, `display_name`  properties will be defined.

Reply:
@Samweli sorry I can't understand, may you rephrase please?

Reply:
@luipir I think we need one place to define the all the algorithms properties not necessary at `data/script.json` we can have a `definitions.py` file instead where we define each alg with its corresponding properties including responsible class name.

for example, algorithm for land cover will have the following definition
 ```
"land-cover": {
        "script version": "1.0.3",
        "description": "Calculate land cover change indicator.",
        "group": "SDG 15.3.1",
        "mode": "local",
        "source": "CalculateLC",
        "display_name": "Land cover (locally)",
        "dialog": "DlgCalculateLC"
    }
```

Reply:
well the old code is bi-valent and it's an architectural problem at the time of refactoring.
Remote scripts are already classified in an external json: data/scripts.json BUT are present only remote scripts (probably associated to the scripts as present in GEE)
Local algorithms (no script concept is associated) are not classified at all and not all processes are local or remote processed... so local processing need a separate structure then remote `script`.
We can unify all, but at cost of a more heavy refactoring (at 6 day of the end of my consultancy). My guideline, with the time available to refactor, was use most of the code with lesser architectural impact... trying to use most of processing code as blackbox and modifying only interface.
I would say "wan't fix" in this phase other than having a clear instruction to do it.